packages/account/src/Sections/Verification/ProofOfOwnership/validation.ts

```diff
@@ -0,0 +1,32 @@
+import { hasInvalidCharacters, isFormattedCardNumber, validFile } from '@deriv/shared';
+import { localize } from '@deriv/translations';
+import { IDENTIFIER_TYPES } from 'Constants/poo-identifier';
+import { TPaymentMethodIdentifier } from 'Types';
+
+export const isValidPaymentMethodIdentifier = (
+    payment_method_identifier: string,
+    identifier_type: TPaymentMethodIdentifier | ''
+) => {
+    const default_error_message = localize('Enter your full card number');
+
+    if (identifier_type === IDENTIFIER_TYPES.CARD_NUMBER) {
+        if (payment_method_identifier.length < 16) {
+            return default_error_message;
+        } else if (payment_method_identifier.length === 16) {
+            return !hasInvalidCharacters(payment_method_identifier) ? null : default_error_message;
+        } else if (payment_method_identifier.length > 19) {
+            return isFormattedCardNumber(payment_method_identifier) ? null : default_error_message;
+        }
```

Comment:
All these number should be from the API, do we have a card to move them to the API?

Reply:
No. There is no card regd any enhancements for POO. The feature is lost in time 🥲